requirements/test.in

```diff
@@ -0,0 +1,14 @@
+# Requirements for test runs
+-c constraints.txt
+
+-r base.txt               # Core dependencies for the cookiecutter
+
+bumpversion==0.5.3
+wheel==0.30.0
+watchdog==0.8.3
```

Comment:
why are we specifying versions for all these libs here?

Reply:
He just took these versions from the old `requirements_dev.txt` and since in the constraints file we use to put a description of why a package is pinned we thought that we could keep them in here untill we are sure why they are pinned.

For that reason in the constraints.txt file he put:

`# TODO: Many pinned dependencies should be unpinned and/or moved to this constraints file.`

Should we move them there? In that case which thing should we put in the description?

Reply:
it'd be ideal if we could remove these constraints and see if tests still pass with the latest versions.
I doubt that all of these libs would need to be constrained.